front/src/views/PatientSignUpView/PatientSignUpView.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+
+import style from './patient_sign_up_view.scss';
+import { Button, TextInput } from '../../components';
+import Svg from 'svg-inline-react';
+import registrationIcon from '../../../assets/icons/user-tie.svg';
+
+class Input extends React.Component {
+  render() {
+    return (
+      <div className={ style['simple-input-box'] }>
+        <h3 className={ style['input-label'] }>
+            { this.props.text }:
+        </h3>
+        <div className={ style['input-container'] }>
+          <div className="row">
+            {
+              // jbc to ten row + col-xs mi sie tu nie podoba trzeba to zmienic
+              // ale inaczej wylazi poza input-container plus jest na 100% szerokosci
+            }
+            <TextInput className="col-xs-12" />
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+export class PatientSignUpView extends Component {
+
+  render() {
+    let buttonStyle = { marginTop: '30px' };
+
+    return (
+      <div className={ style['root'] }>
+        <div className={ style['title-container'] }>
+          <Svg
+            src={ registrationIcon }
+            className={ style['registration-icon'] }
+          />
+          <h1 className={ style['registration-title'] }>
+            Patient registration
+          </h1>
+        </div>
+
+        <div className={ style['user-input-container'] }>
+
+          <Input text="Username" />
+          <Input text="First Name" />
+          <Input text="Last Name" />
+          <Input text="E-mail Address" />
+          <Input text="Password" />
```

Comment:
You have special component PasswordInput. Please don't write two component in one file.  If you want have this in one file I recommend you to use new render function such as renderInputWithLabel(args)
Although naming this component Input doesn't appeal to me. We have something similar to it. TextInput!